File: olo-promotions-sdk-csharp/Olo.Promotions.SDK/Responses/AccruePointsResponse.cs
using Olo.Promotions.SDK.Responses.Models;

namespace Olo.Promotions.SDK.Responses
{
    /// <summary>
    /// Returned for a successful call to the Accrue Points endpoint.
    /// </summary>
    public class AccruePointsResponse
    {
        public Transaction Transaction { get; set; }
    }
}

File: olo-promotions-sdk-go/pkg/promotions/models/responses/accruals/accrue_points_response.go
package accruals

import "olo-promotions-sdk-go/pkg/promotions/models/responses"

// AccruePointsResponse is returned for a successful call to the Accrue Points endpoint.
type AccruePointsResponse struct {
	// Transaction represents the transaction details for the accrued points.
	Transaction responses.Transaction `json:"transaction"`
}
